pwa/src/templates/componentDetail/ComponentsDetailTemplate.tsx

```diff
@@ -61,22 +71,63 @@ export const ComponentsDetailTemplate: React.FC<ComponentsDetailTemplateProps> =
                 {_getComponent.data.embedded.description.longDescription}
               </LeadParagraph>
 
-              <div className={styles.tags}>
+              <div className={styles.layerTags}>
                 <div
                   className={
                     styles[_.camelCase(t(`${_getComponent.data.embedded?.nl.embedded.commonground.layerType}`))]
                   }
                 >
-                  <Tag
-                    label={t(_.upperFirst(_getComponent.data.embedded?.nl.embedded.commonground.layerType))}
-                    icon={<FontAwesomeIcon icon={faLayerGroup} />}
-                  />
-                </div>
-                <div>
-                  {_getComponent.data.developmentStatus && (
-                    <Tag label={_.upperFirst(_getComponent.data.developmentStatus)} />
-                  )}
+                  <ToolTip tooltip="Laag">
+                    <Tag
+                      label={t(_.upperFirst(_getComponent.data.embedded?.nl.embedded.commonground.layerType))}
+                      icon={<FontAwesomeIcon icon={faLayerGroup} />}
+                    />
+                  </ToolTip>
                 </div>
```

Comment:
If we give the `Tooltip` component a `layoutClassName` prop, we could probably get away with not having a wrapper div here.

Also, right now we're setting the `tag-color` and `tag-background-color` within this wrapper div, why not just in the tag itself?